Assets/MRTK/SDK/Features/UX/Scripts/Slate/HandInteractionPanZoom.cs

```diff
@@ -579,16 +617,22 @@ private Vector2 GetUVFromQuadCoord(Vector2 coord)
             float magVertical = (lowerLeft - upperLeft).magnitude;
             float magHorizontal = (upperRight - upperLeft).magnitude;
 
-            float v = Vector2.Dot(coord - upperLeft, (lowerLeft - upperLeft) / (magVertical * magVertical));
-            float h = Vector2.Dot(coord - upperLeft, (upperRight - upperLeft) / (magHorizontal * magHorizontal));
+            if (magVertical != 0.0f && magHorizontal != 0.0f)
```

Comment:
Do you think we'd want to use [`Mathf.Approximately`](https://docs.unity3d.com/ScriptReference/Mathf.Approximately.html) here to account for potential floating point error? Especially since we're comparing to the result of a `.magnitude` call, which returns the result of a square root call:

https://github.com/Unity-Technologies/UnityCsReference/blob/61f92bd79ae862c4465d35270f9d1d57befd1761/Runtime/Export/Math/Vector2.cs#L189

Reply:
Yes, That would be better.